presto-docs/src/main/sphinx/sql/revoke.rst

```diff
@@ -0,0 +1,46 @@
+======
+REVOKE
+======
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    REVOKE [ GRANT OPTION FOR ]
+    (privilege [, ...] | ALL PRIVILEGES)
+    ON TABLE? table_name FROM grantee
+
+Description
+-----------
+
+Revokes the specified privileges from the specified grantee.
+
+Specifying ``ALL PRIVILEGES`` revokes ``DELETE, INSERT`` and ``SELECT`` privileges.
+
+Specifying ``PUBLIC`` revokes privileges from all grantees.
+
+The optional ``GRANT OPTION FOR`` clause also revokes the privileges to grant the specified privileges.
+
+Grant command succeeds provided the grantor possesses the specified privileges as well as the `GRANT OPTION` for those privileges.
+
+Examples
+--------
+
+Revoke ``INSERT`` and ``SELECT`` privileges on the table ``orders`` from user ``alice``::
+
+    GRANT
+    INSERT, SELECT
+    ON orders FROM alice;
+
+Revoke ``SELECT`` privilege on the table ``nation`` from all the users, additionally revoking the privilege to grant ``SELECT`` privilege::
```

Comment:
If Alice has SELECT and GRANT privileges on nation, and I want to revoke GRANT while still allowing her to access the table, do I need to execute two commands?

REVOKE GRANT OPTION FOR 
SELECT
ON nation FROM alice;

GRANT SELECT
ON nation TO alice;